launchpod/angular-launchpod/backend/src/main/java/com/google/launchpod/servlets/FormHandlerServlet.java

```diff
@@ -48,19 +59,39 @@
    */
   @Override
   public void doPost(HttpServletRequest req, HttpServletResponse res) throws IllegalArgumentException, IOException {
-    String podcastTitle = req.getParameter(PODCAST_TITLE);
+    UserService userService = UserServiceFactory.getUserService();
+
+    String title = req.getParameter(TITLE);
     String mp3Link = req.getParameter(MP3_LINK);
-    if (podcastTitle == null || podcastTitle.isEmpty()) {
+    String name = req.getParameter(USER_NAME);
+    String category = req.getParameter(CATEGORY);
+    String email = userService.getCurrentUser().getEmail();
+
+    long timestamp = System.currentTimeMillis();
+    Date date = new Date(timestamp);
+    SimpleDateFormat dateFormat = new SimpleDateFormat("MM/dd/yyyy  HH:mm:ss");
```

Comment:
we should also probably include timezone to the formatted date

Reply:
done.